qos-service-impl/src/main/java/com/palantir/atlasdb/qos/ratelimit/SmoothRateLimiter.java

```diff
@@ -123,7 +121,7 @@ final long reserveEarliestAvailable(int requiredPermits, long nowMicros) {
                 storedPermitsToWaitTime(this.storedPermits, storedPermitsToSpend)
                         + (long) (freshPermits * stableIntervalMicros);
 
-        this.nextFreeTicketMicros = LongMath.saturatedAdd(nextFreeTicketMicros, waitMicros);
+        this.nextFreeTicketMicros = nextFreeTicketMicros + waitMicros;
```

Comment:
nit: This was edited, but not annotated with `CHANGELOG` as other edits are in #2703.

Reply:
i actually re-copied the whole file in that PR, and used a different version that didn't rely on `LongMath`, so this line is no longer edited